pkg/chunkenc/symbols.go

```diff
@@ -50,22 +54,13 @@ func newSymbolizer() *symbolizer {
 	}
 }
 
-// Reset resets all the data and makes the symbolizer ready for reuse
-func (s *symbolizer) Reset() {
-	s.mtx.Lock()
-	defer s.mtx.Unlock()
-
-	s.symbolsMap = map[string]uint32{}
-	s.labels = s.labels[:0]
-	s.size = 0
-	s.compressedSize = 0
-	s.normalizedNames = map[uint32]string{}
-}
```

Comment:
surprised to see this removed? do we not use it? 

Fine with removing it just wanted to double check

Reply:
Yeah, the symbolizer is created per chunk, and since we do not reuse chunk references, we do not reuse symbolizer either. We will have to use a pool to reuse symbolizer, but we will have to see how much it would help since the volume of structured metadata won't be balanced.